acorn/src/options.js

```diff
@@ -91,14 +91,25 @@ export const defaultOptions = {
 
 // Interpret and default an options object
 
+let warnedAboutEcmaVersion = false
+
 export function getOptions(opts) {
   let options = {}
 
   for (let opt in defaultOptions)
     options[opt] = opts && has(opts, opt) ? opts[opt] : defaultOptions[opt]
 
-  if (options.ecmaVersion >= 2015)
+  if (options.ecmaVersion === "latest") {
+    options.ecmaVersion = 1e8
```

Comment:
```suggestion
    options.ecmaVersion = Number.MAX_SAFE_INTEGER
```

Could you utilize `MAX_SAFE_INTEGER` here? Or is the significance to the number `1e8`?

Reply:
That'll make the field a float, potentially making comparisons more expensive. 1e8 fits into a 30-bit integer, and it doesn't seem likely that we'll reach that ecma version ever, so I prefer that.

Reply:
I'd probably even go for something simpler and more obvious as `3000`, but sgtm either way.